lib/CleantalkSP/SpbctWP/Scanner/DBTrigger/DBTriggerModel.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace CleantalkSP\SpbctWP\Scanner\DBTrigger;
+
+class DBTriggerModel
+{
+    /**
+     * Runs the DBTrigger model to find malicious database triggers.
+     * @param array|null $mock_data Mocked triggers for testing purposes.
+     * @return string|true The result of the operation or an error message.
+     * @psalm-suppress PossibleUnusedParameter
+     */
+    public static function run($mock_data = null)
```

Comment:
Using parameter for mock data in live code is a bad way.

Reply:
Fixed https://github.com/CleanTalk/security-malware-firewall/commit/7cbbca42e49e67b143c83f87de2e05c73475b724